packages/data/src/items/actions.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import { apiFetch } from '@wordpress/data-controls';
+
+/**
+ * Internal dependencies
+ */
+import TYPES from './action-types';
+import { NAMESPACE } from '../constants';
+
+export function updateItems( itemType, query, items, totalCount ) {
```

Comment:
Nitpicky and I think we discussed this in one of my data migration PRs:  I like using `set` as a prefix for when we're setting locally and `update` when persisting to the server.

I think it makes it just a bit easier to quickly grok what's happening here.

Reply:
Cool, done in https://github.com/woocommerce/woocommerce-admin/pull/5009/commits/c33c4d4c67d8a3bf483f6bf54051aea5c6e278df